go/common/enclave.go

```diff
@@ -125,8 +125,6 @@ type Enclave interface {
 	StreamL2Updates() (chan StreamL2UpdatesResponse, func())
 	// DebugEventLogRelevancy returns the logs of a transaction
 	DebugEventLogRelevancy(hash gethcommon.Hash) (json.RawMessage, SystemError)
-
-	Config() (*ObscuroEnclaveInfo, SystemError)
```

Comment:
This method existed because there was some config (sequencerID and message bus address) that wasn't available to the host but we wanted to serve those details on obscuroscan API. Since I needed one of them I thought I might as well add both properties and remove the method.

Reply:
I wonder if it might still be useful in the HA setting.
Even as a sanity check to spot config problems early. Basically, when a host starts, or when an enclave is added, the host checks that the config matches the enclave

Reply:
I think we were planning to bake the static network level config into the enclave image which we sign. I agree something like that could be useful but I feel like we haven't reached our end-state of the config situation yet so kinda hard to predict.

Don't feel too strongly either way though, I can put it back if you prefer?